### ISSUE TITLE
Dropdown focus behaviors should be disabled on mobile

### ISSUE THREAD

**Issue by Mammad900** [Home-modules/webapp#3]
These behaviors were intended for desktop users and are an annoyance for mobile uses because of the virtual keyboard activation.
